castle/migrations/0029_challenge.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
I don't understand these files. Is it necessary for them to be checked in? What is their purpose? They seem both important and temporary at the same time.